tests/e2e/navigation/about.wdio-spec.js

```diff
@@ -0,0 +1,47 @@
+const chai = require('chai');
+
+const utils = require('../../utils');
+const commonPage = require('../../page-objects/common/common.wdio.page');
+const loginPage = require('../../page-objects/login/login.wdio.page');
+const aboutPage = require('../../page-objects/common/about.wdio.page');
+
+describe('About page' , async () => {
+  beforeEach(async () => {
+    await loginPage.cookieLogin();
+  });
+
+  it('should open the about page', async () => {
+    await commonPage.goToAboutPage();
+    chai.expect(await (await aboutPage.userName()).getText()).to.equal('admin');
+  });
+
+  it('should display partners', async () => {
+    const partnersDoc = {
```

Comment:
As this represents a doc and is data. Can you move this to a factory object?

Reply:
I'm not a fan of Factory, especially for objects that aren't reused.

Reply:
It introduces an additional layer of unnecessary abstraction where you don't need it to do anything except push a doc to the database.

Reply:
The factory is meant to represent a real life object based on a real config. Not one that meets the needs of the current test. 

It also helps reduce the copy/paste pattern we have in our test code for data. Instead of recreating a hierarchy in every test we just use a factory to do that work.

Reply:
Yes, I understand for hierarchies, but this is one doc.

Reply:
Ok, I gave Rosie a shot :)